common/app/layout/Front.scala

```diff
@@ -122,7 +124,8 @@ case class ContainerCommercialOptions(
   isAdvertisementFeature: Boolean,
   isFoundationSupported: Boolean,
   sponsorshipTag: Option[SponsorshipTag],
-  sponsorshipType: Option[String]
+  sponsorshipType: Option[String],
+  omitMPU: Boolean = false
```

Comment:
Adding an `omitMPU` flag to `CommercialOptions` so that it's easily available in templates.